
<launch>
	
	<rosparam  command="load"  file="$(find pointcloud_painter)/param/pointcloud_painter.yaml"/>
	
	<node
		name    = "pointcloud_painter"
      	pkg     = "pointcloud_painter"
      	type    = "pointcloud_painter"
      	output  = "screen"
  	> 
	</node>

	<node
		name    = "painter_client"
      	pkg     = "pointcloud_painter"
      	type    = "active_painter_demo"
      	output  = "screen"
  	> 
	</node>

	<!-- Static Transform Publishers !-->
	<node
		name    = "front_cam_frame_publisher"   pkg = "tf"   type = "static_transform_publisher"
      	args  	= "-0.295 0.01 1.02 0 1.57 3.14159 		/map  /target_frame 	100"
  	> </node>
	<node
		name    = "rear_cam_frame_publisher"   pkg = "tf"   type = "static_transform_publisher"
      	args  	= "-0.28 0.01 1.02 0 1.57 3.14159  	/map  /front_camera_frame 	100"
  	> </node>
	<node
		name    = "target_frame_publisher"   pkg = "tf"   type = "static_transform_publisher"
      	args  	= "-0.31 0.01 1.02 0 1.57 3.14159  /map  /rear_camera_frame 		100"
  	> </node>

</launch>